src/_data/starters/11ty-SASS.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/NikitaZanella/11ty-SASS",
+	"name": "11ty-SASS",
+	"description": "11ty blog starter template with SASS and Dark Mode",
+	"author": "NikitaZanella",
```

Comment:
Sorry, to confirm. Is this your _Twitter_ handle? I think the 11ty site uses your Twitter handle (not GitHub or other) to resolve avatars.